gamestonk_terminal/options/chains_view.py

```diff
@@ -124,25 +136,26 @@ def display_chains(symbol: str, expiry: str, other_args: List[str]):
         default=-1,
         help="maximum strike price to consider.",
     )
+    parser.add_argument(
+        "-d",
+        "--display",
+        dest="to_display",
+        default=default_columns,
```

Comment:
Instead, I would add here this:
```
type= check_valid_option_chains_headers
```
and define above:
```
def check_valid_option_chains_headers(headers: str) -> str:
    
    columns = [str(item) for item in headers.split(",")]
    
    for header in columns:
        if header not in df_columns:
            raise argparse.ArgumentTypeError("Invalid option chains header selected!")
    
    return columns
```

Reply:
This makes mypy mad since headers is a List ??

Reply:
It should be like this:
```
def check_valid_option_chains_headers(headers: str) -> List[str]:
```